docs/decision-records/wp-cli.md

```diff
@@ -0,0 +1,185 @@
+---
+id: WP-CLI
+title: WP-CLI
+sidebar_label: WP-CLI
+tags: cli, wp-cli, maintenance, security, updates, user-roles, plugin-updates
+---
+
+This document is an overview of the required information that a developer needs to have in order to use WP-CLI in any of the WordPress projects.
+
+# Overview
+
+WP-CLI is a command line interface which allows the users to manage their WordPress websites from the command prompt. To upgrade and install plugins/themes, to generate backups, new posts can be published and most of the regular admin actions can be performed with a set of commands.
+
+The beauty of the WP-CLI is that it gives us direct control over site. Anything that we do in WordPress admin dashboard can be done using WP-CLI instead. 
+
+**Pre requisites**
+- UNIX-like environment (OS X, Linux, FreeBSD, Cygwin); limited support in Windows 
+- PHP 5.6 or later.
+- WordPress 3.7 or later.
+
+## Installation of WP-CLI
+
+**Ubuntu:**
+
+Run the below commands in terminal:
+
+**For Linux:**
```

Comment:
for Linux and Ubuntu it should be same if so can we club its contradicting (looks like missing command for ubuntu)